services/wireguard/endpoint/userspace/tun_windows.go

```diff
@@ -89,7 +89,7 @@ func (tun *nativeTun) Close() error {
 }
 
 func (tun *nativeTun) Flush() error {
-	return tun.Flush()
```

Comment:
We don't need to flush any more?

Reply:
There's no flush on the delegate. This was an error causing stack overflow (calling flush on self)